packages/shared/src/components/notifications/EnableNotification.tsx

```diff
@@ -56,49 +70,48 @@ function EnableNotification({
       'Stay in the loop whenever you get a mention, reply and other important updates.',
   };
   const message = sourceToMessage[source];
-  if (source === NotificationPromptSource.NotificationList) {
-    return (
-      <div className="relative py-4 px-6 w-full bg-theme-float border-l typo-callout border-theme-color-cabbage">
-        <span className="mb-2 font-bold">Push notifications</span>
-        <p className="w-3/5 text-theme-label-tertiary">{message}</p>
-        <span className="flex flex-row gap-4 mt-4">
-          <Button
-            buttonSize="small"
-            className="w-28 text-white btn-primary-cabbage"
-            onClick={requestPermission}
-          >
-            Enable
-          </Button>
-          <Button
-            buttonSize="small"
-            className="w-28 btn-tertiary"
-            onClick={() => setDismissed(true)}
-          >
-            Dismiss
-          </Button>
-        </span>
-        <img
-          className="absolute top-2 right-0"
-          src={cloudinary.notifications.browser}
-          alt="A sample browser notification"
-        />
-      </div>
-    );
-  }
+  const classes = containerClassName[source];
+
   return (
-    <div className="overflow-hidden relative py-4 px-4 mt-3 rounded-16 border typo-callout border-theme-color-cabbage">
-      <p className="w-3/5 text-theme-label-tertiary">{message}</p>
+    <div
+      className={classNames(
+        'overflow-hidden relative py-4 rounded-16 typo-callout',
+        classes,
+      )}
+    >
+      {source === NotificationPromptSource.NotificationList && (
+        <span className="flex flex-row font-bold">
+          {isEnabled && <VIcon className="mr-2" />}
+          {`Push notifications${isEnabled ? ' successfully enabled' : ''}`}
+        </span>
+      )}
+      <p className="mt-2 w-full tablet:w-3/5 text-theme-label-tertiary">
+        {isEnabled ? (
+          <>
+            Changing your{' '}
+            <a
+              className="underline hover:no-underline"
+              href={`${webappUrl}account/notifications`}
+            >
+              notification settings
+            </a>{' '}
+            can be done anytime through your account details
+          </>
+        ) : (
+          message
+        )}
+      </p>
```

Comment:
@el this just looks like there's a lot of changes, but the only difference is the padding/margin/border classes around the container. Also the conditional rendering of the "Push notifications` title.
Then the `p` element content.